doc/classes/Tree.xml

```diff
@@ -123,6 +123,11 @@
 				Returns the column for the currently edited item.
 			</description>
 		</method>
+		<method name="edit_selected">
+			<description>
+				Edits the selected tree item as if it was clicked. The item must be set editable with [method TreeItem.set_editable]. Returns true if the item could be edited. Fails if no item is selected.
```

Comment:
The `<return ...>` type information seems to be missing. Make sure to use `godot --doctool .` to generate the template, not adding the XML manually. See https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html

```suggestion
				Edits the selected tree item as if it was clicked. The item must be set editable with [method TreeItem.set_editable]. Returns [code]true[/code] if the item could be edited. Fails if no item is selected.
```

Reply:
Done.